android/uhabits-android/src/androidTest/java/org/isoron/uhabits/acceptance/steps/CommonSteps.java

```diff
@@ -19,26 +19,40 @@
 
 package org.isoron.uhabits.acceptance.steps;
 
-import androidx.annotation.StringRes;
-import androidx.test.espresso.*;
-import androidx.test.espresso.contrib.*;
-import androidx.test.uiautomator.*;
-import androidx.appcompat.widget.*;
+import android.view.View;
 
+import androidx.annotation.StringRes;
 import androidx.recyclerview.widget.RecyclerView;
-
-import org.isoron.uhabits.*;
+import androidx.test.espresso.PerformException;
+import androidx.test.espresso.contrib.RecyclerViewActions;
+import androidx.test.uiautomator.By;
+import androidx.test.uiautomator.UiSelector;
+import androidx.test.uiautomator.Until;
```

Comment:
To keep our code style consistent, could you please switch these to wildcard imports?